src/cli/commands/config.js

```diff
@@ -16,8 +16,15 @@ export const {run, setFlags} = buildSubCommands('config', {
       return false;
     }
 
-    const [key, val = true] = args;
+    const [key, val] = ((key, val = true) => {
+      if (typeof val === 'string' && (/^(true|false)$/i).test(val)) {
```

Comment:
It won't work for upper/mixed-case letters, and simply using`val.toLowerCase() === 'false' || val.toLowerCase() === 'true'` (or introducing another, intermediate variable to hold the lowercase value) will break on the default `true` value. The typeof check is required.

The regex can be replaced, e.g.

```
if (typeof val === 'string' && (val.toLowerCase() === 'false' || val.toLowerCase() === 'true')) {
```

but it's ugly :/